test/acquisition/test_analytic.py

```diff
@@ -902,11 +910,17 @@ def test_noisy_expected_improvement(self):
             (torch.tensor([[-3.4], [0.8]]), torch.tensor([[0.0], [1.0]])),
             (None, covar_module_2),
         ):
-            self._test_noisy_expected_imrpovement(
-                dtype, use_octf, use_intf, bounds, covar_module
-            )
+            with catch_warnings():
+                simplefilter("ignore", category=NumericsWarning)
+                self._test_noisy_expected_improvement(
+                    dtype=dtype, 
+                    use_octf=use_octf,
+                    use_inf=use_intf,
```

Comment:
This should be a `use_inft`, couldn't test this before adding the merge commit through the web UI.